src/sentry/integrations/github/client.py

```diff
@@ -622,34 +622,27 @@ def get_blame_for_file(
             }}
         }}"""
 
-        contents = self.post(
-            path="/graphql",
-            data={"query": query},
-        )
-
         try:
-            results: Sequence[Mapping[str, Any]] = (
-                contents.get("data", {})
-                .get("repository", {})
-                .get("ref", {})
-                .get("target", {})
-                .get("blame", {})
-                .get("ranges", [])
+            contents = self.post(
+                path="/graphql",
+                data={"query": query},
+                allow_text=False,
```

Comment:
this prevents the wrong type being returned here -- this now *must* be a json response and cannot be a plaintext one